parser-typechecker/package.yaml

```diff
@@ -100,6 +100,7 @@ library:
     - mwc-random
     - NanoID
     - lucid
+    - yaml
```

Comment:
Used for rendering front matter to the top of the file when converting docs to html files.